### DISCUSSION TITLE
timer button looks kinda odd

### DISCUSSION THREAD

**Discussion by Nepp0** [ppy/osu#21761] [Q&A] [answered]
different shape and spacing compared to the other buttons in multiplayer.
![osu_2022-12-22_19-48-03](https://user-images.githubusercontent.com/98234215/209208122-4980443b-551f-476d-aabc-377eb5335a97.jpg)
it was immediately noticeable after i created a lobby so i hope this gets changed.

**Comment by frenzibyte** [comment, marked as answer]
https://github.com/ppy/osu/discussions/21580#discussioncomment-4348324
